src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Input/MouseDevice.cs

```diff
@@ -739,15 +739,15 @@ private object ReevaluateCaptureAsync(object arg)
             // First, check things like IsEnabled, IsVisible, etc. on a
             // UIElement vs. ContentElement basis.
             //
-            if (InputElement.IsUIElement(dependencyObject))
+            if (dependencyObject is UIElement)
             {
                 killCapture = !ValidateUIElementForCapture((UIElement)_mouseCapture);
             }
```

Comment:
```suggestion
            if (dependencyObject is UIElement uie)
            {
                killCapture = !ValidateUIElementForCapture(uie);
            }
```

Please note, _mouseCapture is the dependencyObject at line#736